contrib/msggen/msggen/schema.json

```diff
@@ -79,7 +79,7 @@
         ],
         "properties": {
           "satoshi": {
-            "type": "sat",
+            "type": "msat",
```

Comment:
This doesn't look right.  The psbt code handles satoshi amounts and the description matches this.  Maybe there needs to be a new rust primitive type?

Reply:
see here: https://github.com/ElementsProject/lightning/pull/7230#discussion_r1571096707

Reply:
Ah, thanks for the back story!